README.md

```diff
@@ -184,6 +184,33 @@ The word "Contact" in French is the same so a translated URL is not given, or ne
 - /ru/контакт - Page in Russian
 - /ru/contact - 404
 
+## URL Parameter translation
+
+It's possible to translate URL parameters by listening to the translate.params.translateParams event, which is fired when switching languages.
+
+```php
+Event::listen('translate.params.translateParams', function($page, $params, $oldLocale, $newLocale) {
+    if ($page->baseFileName == 'your-page-filename') {
+        return YourModel::translateParams($params, $oldLocale, $newLocale);
+    }
+});
+```
+
+In YourModel, one possible implementation might look like this:
+
+```php
+public static function translateParams($params, $oldLocale, $newLocale) {
+    $newParams = $params;
+    foreach ($params as $paramName => $paramValue) {
+        $records = self::transWhere($paramName, $paramValue, $oldLocale)->first();
```

Comment:
Just thought of this now @munxar, thanks for your patience with this. Is this an optimal query? I feel like this could cause a lot of extra queries to be run, one for every URL parameter. What are your thoughts on that? It's not a deal breaker for the PR but I'd like to try and provide as good as an example as possible.

Reply:
@LukeTowers I think that even with 10 URL parameters it shouldn't matter, there just read queries. If performance is really an issue the implementer should cache the results. (not to mention the DB usually has a query cache anyways...)